src/components/views/elements/TooltipTarget.tsx

```diff
@@ -44,6 +44,17 @@ const TooltipTarget: React.FC<IProps> = ({
     const show = () => setIsVisible(true);
     const hide = () => setIsVisible(false);
 
+    const tooltip = isVisible && <Tooltip
+        id={id}
+        className={className}
+        tooltipClassName={tooltipClassName}
+        label={label}
+        yOffset={yOffset}
+        alignment={alignment}
+        visible={isVisible}
+        maxParentWidth={maxParentWidth}
+    />;
```

Comment:
Don't add DOM elements to the page for hidden tooltips.

This is the same pattern that we use in [`src/components/views/elements/AccessibleTooltipButton.tsx#L75-L80`](https://github.com/matrix-org/matrix-react-sdk/blob/31f0a37ca2eeba6a6296787f2fcb33c4b26efebc/src/components/views/elements/AccessibleTooltipButton.tsx#L75-L80)

---

Previously, we just had a bunch of hidden `This room is public` tooltips from the decorated avatars in the room list, [`src/components/views/avatars/DecoratedRoomAvatar.tsx#L190-L196`](https://github.com/matrix-org/matrix-react-sdk/blob/31f0a37ca2eeba6a6296787f2fcb33c4b26efebc/src/components/views/avatars/DecoratedRoomAvatar.tsx#L190-L196), even though no tooltips were actually being shown on the page.

<img width="354" alt="" src="https://user-images.githubusercontent.com/558581/156284824-3acf3985-fcf3-43b4-8810-3712f78cf550.png">